ArduPlane/Parameters.cpp

```diff
@@ -142,12 +142,28 @@ const AP_Param::Info Plane::var_info[] = {
 
     // @Param: TKOFF_THR_MAX_T
     // @DisplayName: Takeoff throttle maximum time
-    // @Description: This sets the time that maximum throttle will be forced during a fixed wing takeoff without an airspeed sensor. If an airspeed sensor is being used then the throttle is set to maximum until the takeoff airspeed is reached.
+    // @Description: This sets the time that maximum throttle will be forced during a fixed wing takeoff.
```

Comment:
This has been made common to both with- and without-airspeed cases. I believe it makes it easier to tell the various takeoff combinations apart, without negatively affecting the with-airspeed takeoffs.
The general idea is that they should be max-throttle by default anyways.